yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/SignedTimestampValue.java

```diff
@@ -0,0 +1,24 @@
+package com.yoti.api.client.spi.remote;
+
+import com.yoti.api.client.DateTime;
+import com.yoti.api.client.SignedTimestamp;
+
+public class SignedTimestampValue implements SignedTimestamp {
```

Comment:
`DefaultSignedTimestamp` or `SimpleSignedTimestamp`? Do we have other implementation of this interface? SignedTimestampValue seems a strange name.

Reply:
It follows the pattern in place for _DateValue implements Date_.